InstructorStudentListPageUiTest fails at line 172

This happens regularly on Travis and sometimes on local environments as well.

The stack trace is:
```
java.lang.AssertionError: expected:<http://localhost:8888/page/studentProfilePic?studentemail=F702AF37C82846594241AA61DA9E8121513646F6B96E433CF5475D7A0DDC960D8A760297A033C316073F728DF8BAE432&courseid=C179779D673504E85F7ADD572ED9CD4D&user=instructorWith2Courses> but was:<http://localhost:8888/images/profile_picture_default.png>
        at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
        at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
        at teammates.test.pageobjects.InstructorStudentListPage.verifyProfilePhoto(InstructorStudentListPage.java:117)
        at teammates.test.cases.ui.browsertests.InstructorStudentListPageUiTest.testShowPhoto(InstructorStudentListPageUiTest.java:172)
        at teammates.test.cases.ui.browsertests.InstructorStudentListPageUiTest.testAll(InstructorStudentListPageUiTest.java:68)
```

--- a/src/test/java/teammates/test/pageobjects/InstructorStudentListPage.java
+++ b/src/test/java/teammates/test/pageobjects/InstructorStudentListPage.java
@@ -114,7 +114,8 @@ public class InstructorStudentListPage extends AppPage {
                                          .findElement(By.cssSelector(".profile-pic-icon-click > img"));
         Actions action = new Actions(browser.driver);
         action.click(photo).build().perform();
-        assertEquals(profilePhotoSrc, browser.driver.findElement(By.cssSelector(".popover-content > .profile-pic"))
+        assertEquals(profilePhotoSrc, browser.driver.findElement(By.id("studentphoto-c" + rowId))
+                                                    .findElement(By.cssSelector(".popover-content > .profile-pic"))
                                                     .getAttribute("src"));
     }
 

